content/en/_redirects

```diff
@@ -234,3 +233,11 @@ docs/started/requirements/                    /docs/started/getting-started/
 /docs/distributions/gke/*           https://googlecloudplatform.github.io/kubeflow-gke-docs/docs/:splat
 /docs/distributions/ibm/*           https://ibm.github.io/manifests/
 /docs/distributions/nutanix/*       https://nutanix.github.io/kubeflow-manifests/
+
+# Refactor Katib docs
+/docs/components/hyperparameter                /docs/components/katib/overview
+/docs/components/katib/hyperparameter          /docs/components/katib/getting-started.md
```

Comment:
@andreyvelich this leads to a 404, because you have put `/docs/components/katib/getting-started.md` (with `.md`)

Reply:
Great catch, I fixed it: https://github.com/kubeflow/website/pull/3733